app/index.js

```diff
@@ -198,32 +199,48 @@ module.exports = generators.Base.extend({
     }.bind(this));
   },
 
-  copyTemplates: function () {
-    this.fs.copyTpl(
-      this.templatePath(version + '/_package.json'),
-      this.destinationPath(folderPath + 'package.json'),
-      {
-        slugifiedAppName: this.slugifiedAppName,
-        appDescription: this.appDescription,
-        capitalizedAppAuthor: this.capitalizedAppAuthor
-      });
-    this.fs.copyTpl(
-      this.templatePath(version + '/_bower.json'),
-      this.destinationPath(folderPath + 'bower.json'),
-      {
-        slugifiedAppName: this.slugifiedAppName,
-        appDescription: this.appDescription
-      });
-    this.fs.copyTpl(
-      this.templatePath(version + '/config/env/_default.js'),
-      this.destinationPath(folderPath + 'config/env/default.js'),
-      {
-        appName: this.appName,
-        appDescription: this.appDescription,
-        appKeywords: this.appKeywords
-      });
+  replaceConfigs: function () {
+    var done = this.async();
+
+    var packageJson = JSON.parse(fs.readFileSync(folderPath + 'package.json'));
```

Comment:
Take a look at http://yeoman.io/authoring/file-system.html and https://github.com/sboudrias/mem-fs-editor because I think we can just use `this.fs.readJSON`